### ISSUE TITLE
PDF Exporting an underscore line under the Title

### ISSUE THREAD

**Issue by ZYN-NOVUS** [HappenApps/Quiver#770]
Is there a particular style to remove this annoying underscore line below the Title?  The underscore doesn't appear in HTML, just in PDF.  It only appears below the Title following each Title Page.

![screen shot 2017-01-18 at 6 58 00 pm](https://cloud.githubusercontent.com/assets/25213918/22090620/13c33bc2-ddb0-11e6-897d-b80f7c7313cb.png)

**Comment by kpmansfield** [comment]
Agree that this is annoying - finding the same occurs when exporting to PDF on macOS 10.12.3. Would be grateful for a fix. Otherwise Quiver is great.

**Comment by ZYN-NOVUS** [comment]
I was unsuccessful in finding a solution to removing that annoying bar.  But, I still believe there is a solution by modifying #titlearea in some way.  I just haven't found it yet.  However, I have a workaround if you're interested.  

**Caution:**  Before attempting this, I suggest you save any CSS settings you have made before in PDF and Print as this will override #titlearea and h1 for all future PDF's and Printouts.  

Under Preferences, Styles, PDF and Print, copy and paste the following CSS code under Text cell comment.

```
#titlearea{
  visibility: hidden; 
}

#contentarea {
  margin-top: -80px;
}

h1 {
  border-bottom: 2px solid #fff;   /* change #fff if you don't want white bar */
  margin-bottom: 2px;
  padding-bottom: 10px;
}

```
This code removes the title area and lets you customize your own title in the content area.  Whatever you want as a title, use h1 and it will appear similar to the previous title area.  

**Note:**  This only affects PDF and Print, so if you wish for it appear in the preview and editor, then copy the CSS code there as well.

**Comment by ylian** [comment]
Please don't bother with these solutions. It's a simple bug and I already fixed it in the coming update. I have been testing the update over the weekend and will push out shortly.

**Comment by ylian** [comment]
Of course, if you really need a solution right now, you can put this into the css file for PDF/print (Preferences > Styles > Edit CSS for PDF/Print):
```css
#tag-field {
  display: none;
}
```

**Comment by kpmansfield** [comment]
Many thanks both for the response—appreciated.

**Comment by ylian** [comment]
Fixed in 3.0.4.
